openapi/single-sign-on/single-sign-on.yaml

```diff
@@ -0,0 +1,357 @@
+          description: 'Client ID in Vendasta''s OpenID. This is associated with a service provider, which determines the destination.'
+        - schema:
+            type: string
+          in: query
+          name: account_id
+          description: Business Location ID (Optional Service Context)
+        - schema:
+            type: string
+          in: query
+          name: group_path
+          description: 'Group ID (Optional Service Context) '
+        - schema:
+            type: string
+            enum:
+              - none
+              - login
+              - consent
+              - select_account
+          in: query
+          name: prompt
+          description: Prompt controls what prompts will or will not be shown to the user. See details...
+        - schema:
+            type: string
+          in: query
+          name: redirect_uri
+          description: Redirect URI is where the User will be sent after the Authorization flow with either the result or an error.
+        - schema:
+            type: string
+            example: code
+          in: query
+          name: response_type
+          required: true
+          description: The Open ID authorization response type. Only 'code' is supported.
+      tags:
+        - Single-Sign-On
+    parameters:
+      - schema:
+          type: string
+        name: partnerID
+        in: path
+        required: true
+        description: Partner ID providing identity namespacing
+  '/{partnerID}/oauth2/logout':
+    get:
+      summary: OpenID Logout
+      operationId: openid-logout
+      responses:
+        '200':
+          description: OK
+      x-lifecycle:
+        status: trustedTester
+      description: |-
+        [Status](https://developers.vendasta.com/platform/ZG9jOjEwMTU2NTYy-versioning): `Trusted Tester`
+
+        Browsers can be directed to this endpoint to logout. This will end the User's Session at Single-Sign-On and Sessions in other services which were accessed using the Single-Sign-On Session.
+
+        This operation has no security necessarily, but logout can only act upon a browser's session via the request's `vig_session` cookie.
+      parameters:
+        - schema:
+            type: string
+          in: query
+          name: post_logout_redirect_uri
+          description: 'URI to redirect to after logout. [This must be preregistered (spec), but we do not yet require or allow this]. [spec miss: id_token_hint is required when this parameter is used.]'
+        - schema:
+            type: string
+          in: query
+          name: namespace
+          description: 'Used as the partnerID if the endpoint is called without a partnerID in the path (`/oauth2/token`), and is required when doing so.'
+        - schema:
+            type: string
+          in: cookie
+          name: vig-namespaced-session
+          description: 'Cookie for a Vendasta Identity Gateway session, stored at namespaced paths.'
+          required: true
```

Comment:
This is definitely "inner working" territory that isn't something to be built upon and could change, but I could see this helping in understanding the logout process and doesn't really give anything away.